app/templates/events/view.hbs

```diff
@@ -38,16 +38,32 @@
 
   <div class="row" style="padding-top: 15px">
     <div class="sixteen wide column">
-      <div class="ui fluid pointing stackable menu {{unless device.isMobile 'secondary'}}">
-        {{#link-to 'events.view.index' class='item'}}
-          {{t 'Overview'}}
-        {{/link-to}}
-        <a href="#" class='item'>{{t 'Tickets'}}</a>
-        <a href="#" class='item'>{{t 'Scheduler'}}</a>
-        <a href="#" class='item'>{{t 'Sessions'}}</a>
-        <a href="#" class='item'>{{t 'Speakers'}}</a>
-        <a href="#" class='item'>{{t 'Export'}}</a>
-      </div>
+      {{#if device.isMobile}}
```

Comment:
We have a lot of tabs. Can you make this into a component so that it can be reused everywhere ?